home/models.py

```diff
@@ -1,3 +1,24 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Subject(models.Model):
+    subject_name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.subject_name
+
+
+class Sub_Subject(models.Model):
+    sub_subject_name = models.CharField(max_length=100)
+    related_subject = models.ForeignKey(Subject, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.sub_subject_name
+
+
+class Book(models.Model):
+    book_name = models.CharField(max_length=100)
+    related_subject = models.ForeignKey(Subject, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.book_name
```

Comment:
A couple of comments. 
If this is what you actually want, then it is ok as is.
* This structure only allows for one level of sub_subject
* Book can only be tied to a subject and not a sub_subject
* If you don't want to attach a sub subject to a book, what do you plan on doing with the table?
* Can a book have multiple subjects? (according to this design, it can't)

Reply:
Yes, we planned on having sub-subjects for each subject. (just one level)
Books will be tied to subject, and only one.
And the user's questions in the app will be categorized in certain subject, and sub-subject, with the option to add a book reference with the same subject.